src/hevm/src/EVM/Types.hs

```diff
@@ -434,17 +434,17 @@ readN s = fromIntegral (read s :: Integer)
 readNull :: Read a => a -> String -> a
 readNull x = fromMaybe x . Text.Read.readMaybe
 
-wordField :: JSON.Object -> Text -> JSON.Parser W256
+wordField :: JSON.Object -> Key -> JSON.Parser W256
```

Comment:
Is this related to an aeson version bump?

Reply:
yes